scripts/prepare-dual-dgx-station.mts

```
@@ -0,0 +1,1055 @@
+    const directoryFd = fs.openSync(directory, fs.constants.O_RDONLY);
+    try {
+      fs.fsyncSync(directoryFd);
+    } finally {
+      fs.closeSync(directoryFd);
+    }
+  } finally {
+    if (fd !== null) fs.closeSync(fd);
+    try {
+      fs.unlinkSync(temporary);
+    } catch (error) {
+      if ((error as NodeJS.ErrnoException).code !== "ENOENT") throw error;
```

Biome (2.5.3)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)